### PR TITLE
Added backend curl-options

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/doctrine/DoctrinePHPCRBundle/pull/251

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@dbu you have mentioned a `versionadded` construct. what do you mean exactly?

File: bundles/phpcr_odm/configuration.rst [hunk 1]
```diff
@@ -179,6 +179,13 @@ severe performance penalty.
 Whether to automatically update nodes having ``mix:lastModified``.
 See `last modified listener cookbook entry`_.
 
+``backend curl_options``
+""""""""""""""""""""""
```
Comment (dantleech) [inline bundles/phpcr_odm/configuration.rst]: Number of underline characters must match the length of the title.

Comment (wachterjohannes) [inline bundles/phpcr_odm/configuration.rst]: :+1:

**Comment by dbu** [comment]
@wachterjohannes look for "versionadded" in the file you are editing, you should find some examples of how this looks

**Comment by wachterjohannes** [comment]
ok i see should i add versionadded 1.3?

**Pull request merged**